src/Centreon/Domain/PlatformInformation/UseCase/V20/UpdatePartiallyPlatformInformation.php

```diff
@@ -186,17 +185,17 @@ private function updateProxyOptions(Information $proxyInformation, ?string $cent
     /**
      * Update the Platform Type
      *
-     * @param PlatformInformation $platformInformationUpdate
+     * @param PlatformInformation $platformInformationToUpdate
      * @param PlatformInformation $currentPlatformInformation
      * @throws RemoteServerException
      */
-    private function updatePlatformTypeOrFail(
-        PlatformInformation $platformInformationUpdate,
+    private function updatePlatformType(
```

Comment:
the method is too much specific to call it updatePlatformType. Maybe convertCentralToRemote suits it best

Reply:
depending of the PlatformInformation to update and the current PlatformInformation, this method could convert Central to Remote or a Remote to Central.

Reply:
the intention is not reflected in the method name... As you will not be able to change the type or platforms which are not remote or central (eg : poller and coming MBI / MAP / deportedDB / centreon-cloud)

Reply:
agreed, we should discuss further on how to name this method with a relevant name that is not too long.